### ISSUE TITLE
[Topic] WCAG and compliance

### ISSUE THREAD

**Issue by rianrietveld** [wpaccessibility/wp-a11y-docs#97]
URL: https://wpaccessibility.org/docs/topics/wcag/
Code: https://github.com/wpaccessibility/wp-a11y-docs/blob/main/docs/topics/wcag/index.md

Please read [How to contribute](https://wpaccessibility.org/docs/contribute/) before you start.

## Tasks
This content should be reviewed and expanded and maybe divided into separate topics.
Do we have a baseline for WordPress? See also "Discussion" on https://github.com/wpaccessibility/wp-a11y-docs/issues/92.
